.github/workflows/release.yaml

```diff
@@ -133,7 +133,7 @@ jobs:
           --path="./flux-system" \
           --source=${{ github.repositoryUrl }} \
           --revision="${{ github.ref_name }}/${{ github.sha }}"
-      - uses: sigstore/cosign-installer@7cc35d7fdbe70d4278a0c96779081e6fac665f88 # v2.8.0
+      - uses: sigstore/cosign-installer@b6757d8360bb6b9803c38b68e8cb7442baaf7eb5 # v2.8.0
```

Comment:
@pjbgf there is no cosign-installer release with this hash, what is dependabot doing here? The latest version should be v2.8.1 but the comment says 2.8.0...

Reply:
Seems like it is tracking against `main` instead of version/version range.

Maybe because the previous hash was not aligned with the comment?

Reply:
I'm not sure why it did this, but I think we need to rollback and pin it to semver somehow.

Reply:
Created a PR to revert this and linked to this one.